Since v1.3.5 I cannot build httpx on termux/android

### httpx version:
v1.3.5

### Current Behavior:
Since version 1.3.5, I have been unable to successfully build httpx on Termux for Android.

### Expected Behavior:
v1.3.4 build normally.

### Steps To Reproduce:
```
~/test $ go install -v github.com/projectdiscovery/httpx/cmd/httpx@latest

github.com/projectdiscovery/utils/permission
# github.com/projectdiscovery/utils/permission
../../go/pkg/mod/github.com/projectdiscovery/utils@v0.0.54/permission/permission_other.go:10:6: checkCurrentUserRoot redeclared in this block                                                                 
../../go/pkg/mod/github.com/projectdiscovery/utils@v0.0.54/permission/permission_linux.go:16:6: other declaration of checkCurrentUserRoot                                                     
../../go/pkg/mod/github.com/projectdiscovery/utils@v0.0.54/permission/permission_other.go:15:6: checkCurrentUserCapNetRaw redeclared in this block                                                            
../../go/pkg/mod/github.com/projectdiscovery/utils@v0.0.54/permission/permission_linux.go:21:6: other declaration of checkCurrentUserCapNetRaw
```

To avoid this build error, I need to downgrade to version 1.3.4.

```
go install -v github.com/projectdiscovery/httpx/cmd/httpx@v1.3.4
```

I am experiencing the same issue with other ProjectDiscovery tools such as naabu and uncover.

--- a/go.mod
+++ b/go.mod
@@ -18,7 +18,7 @@ require (
 	github.com/projectdiscovery/goconfig v0.0.1
 	github.com/projectdiscovery/goflags v0.1.24
 	github.com/projectdiscovery/gologger v1.1.11
-	github.com/projectdiscovery/hmap v0.0.22
+	github.com/projectdiscovery/hmap v0.0.23
 	github.com/projectdiscovery/mapcidr v1.1.12
 	github.com/projectdiscovery/rawhttp v0.1.21
 	github.com/projectdiscovery/retryablehttp-go v1.0.31
@@ -48,7 +48,7 @@ require (
 	github.com/projectdiscovery/fastdialer v0.0.39
 	github.com/projectdiscovery/ratelimit v0.0.9
 	github.com/projectdiscovery/tlsx v1.1.5
-	github.com/projectdiscovery/utils v0.0.58
+	github.com/projectdiscovery/utils v0.0.61-0.20231101202412-3b40c4fc4e04
 	github.com/stretchr/testify v1.8.4
 	github.com/zmap/zcrypto v0.0.0-20230422215203-9a665e1e9968
 	go.uber.org/multierr v1.11.0
@@ -96,7 +96,7 @@ require (
 	github.com/mattn/go-isatty v0.0.19 // indirect
 	github.com/mattn/go-runewidth v0.0.14 // indirect
 	github.com/mholt/archiver v3.1.1+incompatible // indirect
-	github.com/minio/selfupdate v0.6.0 // indirect
+	github.com/minio/selfupdate v0.6.1-0.20230907112617-f11e74f84ca7 // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/muesli/reflow v0.3.0 // indirect

--- a/go.mod
+++ b/go.mod
@@ -48,7 +48,7 @@ require (
 	github.com/projectdiscovery/fastdialer v0.0.39
 	github.com/projectdiscovery/ratelimit v0.0.9
 	github.com/projectdiscovery/tlsx v1.1.5
-	github.com/projectdiscovery/utils v0.0.61-0.20231101202412-3b40c4fc4e04
+	github.com/projectdiscovery/utils v0.0.61
 	github.com/stretchr/testify v1.8.4
 	github.com/zmap/zcrypto v0.0.0-20230422215203-9a665e1e9968
 	go.uber.org/multierr v1.11.0

